### PR TITLE
Merge point parse & explode

### PR DESCRIPTION
**Pull request by benbjohnson**

### CODE REVIEW

**Comment by benbjohnson** [comment]
Needs to rebase after https://github.com/influxdata/influxdb/pull/12391 is merged.

**Comment by e-dard** [comment]
Did this get closed and opened again? I thought there was already a bunch of review done?

**Comment by benbjohnson** [comment]
@e-dard Previous discussion here: https://github.com/influxdata/influxdb/pull/11617

I rebased and updated this PR.

File: models/points.go [hunk 1]
```diff
@@ -289,13 +288,23 @@ const (
...
 // with each point separated by newlines.  If any points fail to parse, a non-nil error
 // will be returned in addition to the points that parsed successfully.
-func ParsePoints(buf []byte) ([]Point, error) {
-	return ParsePointsWithPrecision(buf, time.Now().UTC(), "n")
+func ParsePoints(buf, mm []byte) ([]Point, error) {
+	return ParsePointsWithPrecision(buf, mm, time.Now().UTC(), "n")
+}
+
+// ParsePointsV1 parses points in buf but does not rewrite the measurement/field.
+func ParsePointsV1(buf, mm []byte) ([]Point, error) {
```
[the remark this answers is not in the document]
Comment (benbjohnson) [reply models/points.go]: These were being used during transition but they're no longer necessary. Removed.

**Pull request merged**